go/private/actions/stdlib.bzl

```diff
@@ -63,24 +63,46 @@ def _should_use_sdk_stdlib(go):
             not go.mode.gc_goopts and
             go.mode.link == LINKMODE_NORMAL)
 
-def _build_stdlib_list_json(go):
+def _build_stdlib_list_json(go, env):
     out = go.declare_file(go, "stdlib.pkg.json")
     args = go.builder_args(go, "stdliblist")
     args.add("-sdk", go.sdk.root_file.dirname)
     args.add("-out", out)
+
     go.actions.run(
         inputs = go.sdk_files,
```

Comment:
If cgo is enabled, you also need to pass `go.crosstool` as input

Reply:
I was going to help set up a hermetic cc toolchain for you to reproduce the error we see internally, but found that the CI of this PR is alway failing with the same issue

Reply:
This suggestion fixes the c++ issue!